src/posts/shipping-react-native-apps-with-fastlane.md

```diff
@@ -19,7 +19,7 @@ That's when [**Fastlane**](https://fastlane.tools) comes to the rescue ⛑! In t
 Before [installing Fastlane](https://github.com/fastlane/fastlane#installation) make sure you have the latest Xcode command line tools installed, then install Fastlane ⬇️.
 
 ```bash
-$ brew cask install fastlane
+$ brew install fastlane --cask
```

Comment:
Hey! According to the official docs `--cask` is not needed anymore, right?

See: https://docs.fastlane.tools

<img width="360" alt="CleanShot 2023-09-16 at 20 25 42@2x" src="https://github.com/carloscuesta/carloscuesta.me/assets/7629661/0fec5df1-cc8d-441f-8181-8158a6f25e00">

Reply:
that's right actually. I've corrected accordingly.